### PR TITLE
Feat: Footer Section

### PR DESCRIPTION
**Pull request by Ntwali-Josue**
## Describe your changes

- Added a responsive `three-columned` footer on `desktop` and `tablet` screens.

## Issue ticket number and link
- ID: #016
- Link: https://www.notion.so/apeunit/Footer-Section-35b1b0ea03b044b59dfe14a3792d9e40

## Tasks completed
- [x] Implement Footer on all screens
 
## Tasks not completed
- None

### CODE REVIEW

File: components/layout/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import Link from "next/link";
+import Anchor from "../ui/Anchor";
+import anchors from "../../data/footerAnchor";
+
+export default function Footer() {
+  return (
+      <footer className="flex mx-10 text-white flex-col md:flex-row  md:justify-between  mb-12 mt-17.25  md:mt-31.75 xl:mt-33.75">
+        <p>
```
Comment (Delyc) [inline components/layout/Footer.jsx:8]: This text here has some space between which you didn't include.

Yours:
image

Figma:

image


There is a gap of 10px, and it should looks exactly like this one below:

image

I think the way you implemented the last one, could work on the above also to have the exact result as on the design

File: components/layout/Footer.jsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import Link from "next/link";
+import Anchor from "../ui/Anchor";
+import anchors from "../../data/footerAnchor";
+
+export default function Footer() {
+  return (
+      <footer className="flex mx-10 text-white flex-col md:flex-row  md:justify-between  mb-12 mt-17.25  md:mt-31.75 xl:mt-33.75">
```
Comment (Delyc) [inline components/layout/Footer.jsx:7]: This mx-10 only applies on desktop. Add xl to mitigate it from affecting mobile as it is making the content fall too inside that it should normally be.

image

Footer content should be on the same like as the word "cooperate".

**Review by Delyc — CHANGES_REQUESTED** [review]

File: components/layout/Footer.jsx (hunk 1 above)
Comment (Ntwali-Josue) [reply components/layout/Footer.jsx:8]: Sure, Will do as requested.

**Review by Delyc — APPROVED** [review]
Well done!

**Review by mhenriette — APPROVED** [review]
looks good !!

**Pull request merged**